src/current/_includes/releases/v24.1/v24.1.1.md

```diff
@@ -0,0 +1,103 @@
+
+- Fixed a bug that was present since v22.2 where [changefeeds]({% link v24.1/change-data-capture-overview.md %}) with long-running [initial scans]({% link v24.1/create-changefeed.md %}#initial-scan) might incorrectly restore checkpoint job progress and drop events during [changefeed restarts]({% link v24.1/changefeed-messages.md %}#duplicate-messages) due to transient errors or node restarts. The bug was most likely to occur in clusters with the following contributing factors:
+    - The `changefeed.shutdown_checkpoint.enabled` [cluster setting]({% link v24.1/cluster-settings.md %}) was enabled (in clusters running v23.2 and later).
+    - The cluster settings `changefeed.frontier_checkpoint_frequency` and `low changefeed.frontier_highwater_lag_checkpoint_threshold` were set low, which resulted in the initial scan taking many multiples of the configured frequency to complete.
+    - There were multiple target tables with significant differences in row counts in one changefeed.
+    - The changefeed target tables were large with many ranges.
+    - The initial scan took a long time to complete (an hour or longer). [#123967][#123967]
+- History retention jobs created upon completion of [cluster replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) no longer erroneously indicate that they failed when they expire. [#124055][#124055]
+
+<h3 id="v24-1-1-sql-language-changes">SQL language changes</h3>
+
+- The  [optimizer]({% link {{ page.version.version }}/cost-based-optimizer.md %}) can now plan constrained scans over partial indexes in more cases, particularly on partial indexes with predicates referencing [virtual computed columns]({% link {{ page.version.version }}/computed-columns.md %}#virtual-computed-columns). [#123468][#123468]
+- The storage parameter `ttl_delete_rate_limit`, which determines the rate limit for deleting expired rows, is now set to `100` by default. [#124354][#124354]
+- CockroachDB no longer limits precision when converting [spatial data types]({% link {{ page.version.version }}/architecture/glossary#data-types.md %}) to JSON. [#124536][#124536]
+- The `optimizer_push_offset_into_index_join` [session setting]({% link {{ page.version.version }}/session-variables.md %}) has been added. When enabled, the  [optimizer]({% link {{ page.version.version }}/cost-based-optimizer.md %}) will attempt to push offset expressions into index join expressions to produce more efficient query plans. The setting is enabled by default on v24.1 and later, and disabled on v23.2. [#124490][#124490]
+- The default value of the `sql.defaults.results_buffer.size` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) has been changed from 16KiB to 512KiB. This reduces the chance that clients using [`READ COMMITTED`]({% link {{ page.version.version }}/read-committed.md %}) transactions will hit errors that cannot automatically be retried within CockroachDB. [#124633][#124633]
```

Comment:
```suggestion
- The default value of the `sql.defaults.results_buffer.size` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) has been changed from 16KiB to 512KiB. This reduces the chance that clients using [`READ COMMITTED`]({% link {{ page.version.version }}/read-committed.md %}) transactions will encounter errors that cannot automatically be retried within CockroachDB. [#124633][#124633]
```
Is "within CockroachDB" redundant?

Reply:
Hmm IMO it clarifies that the automatic retries are meant to be performed by the DB rather than by the client.